NineMensMorris/include/splash.h

```diff
@@ -0,0 +1,21 @@
+#ifndef SPLASH_H
+#define SPLASH_H
+
+#include <QGraphicsScene>
+#include <QGraphicsView>
+#include <QImage>
+#include <QThread>
```

Comment:
Could you just do a quick push to get rid of the QImage and QThread includes?

Reply:
Ok, I believe they've been removed now.